### ISSUE TITLE
Time format

### ISSUE THREAD

**Issue by linhvu1** [NTIA/scos-sensor#99]
It would be nice to have next_task_time, start, stop time shown in hhmmss format (similar to duration). It is easier for user to know when the acquisition will start or stop.

**Comment by djanderson** [comment]
I need a quick review of the above proposal because it would mean a change to the `SchedulEntry` API. @MichaelCotton , @bradeales . Thoughts?

**Comment by MichaelCotton** [comment]
I like Linh's suggestion to drop unix time.
If null = blank, then 1-5 is good logic and straightforward.
1-5 should make it to SCOS-Transfer-Spec. If any is left out, there is room for interpretation and confusion.

**Comment by bradeales** [comment]
What about leaving unix time, but have a dynamically updating natural language sentence below, clarifying what's going to happen?

E.g. "The sensor will start acquiring immediately for 10 seconds, repeating this acquisition every 5 minutes, for the next 30 days."

**Comment by djanderson** [comment]
@MichaelCotton good catch, I meant "unspecified" instead of "null". So the updated rules would be:

1) `start` is unspecified (meaning start ASAP) or a string in the datetime string format we currently use
2) `relative_stop` is unspecified or a number of seconds after start
3) `absolute_stop` is unspecified or a datetime string
4) specifying both `relative_stop` and `absolute_stop` is an error
5) leaving *both* `relative_stop` and `absolute_stop` unspecified mean "never stop" (same as leaving `stop` unspecified now)

**Comment by djanderson** [comment]
@bradeales that would be a pretty cool aide, but I think *having* to have that kind of thing indicates that the API isn't clear, so I think the proposed changes are a lower-level improvement that still doesn't preclude API consumers from doing something like that at a higher level.

**Comment by MichaelCotton** [comment]
Unix time v ISO 8601. I am happy we are having this debate bc I feel strongly that we need to agree upon one choice for the tfr spec. 

Pros for unix time: timezone agnostic 
Pros for ISO 8601: consistent w SigMF, intuitive to user

**Comment by djanderson** [comment]
The ISO 8601 spec as used by SigMF is pinned to zulu time so timezones should not be an issue. All instances of ISO 8601 datetime strings that I use follow this convention as well. From their spec:

```
The datetime Pair

This name/value pair must be an ISO-8601 string, as defined by RFC 3339, where the only allowed time-offset is Z, indicating the UTC/Zulu timezone.
```

**Comment by djanderson** [comment]
I don't see any major downsides to switching to ISO 8601 datetime format. It's a simple conversion to unix timestamps at the serializer so this change doesn't even require a database model change for me. I'm in favor.

**Comment by bradeales** [comment]
Another solution (but I suspect a lot more work) would be to combine `relative_stop` and `absolute_stop` into a single `stop` parameter. If you enter an integer, it's interpreted as relative; where as entering datetime would be absolute.
